index.d.ts

```diff
@@ -721,7 +721,7 @@ declare namespace Eris {
     frameSize?: number;
     sampleRate?: number;
   }
-  type PossiblyUncachedMessage = Message | { id: string; channel: TextableChannel };
+  type PossiblyUncachedMessage = Message | { id: string; channel: TextableChannel } | { id: string; channel: { id: string } };
```

Comment:
```suggestion
  type PossiblyUncachedMessage = Message | { id: string; channel: TextableChannel | { id: string } };
```

Please double check this on VSC i am suggesting this on Github without having my morning cup of tea 🍵